app/models/folio/patron.rb

```diff
@@ -125,76 +132,68 @@ def remaining_checkouts
       borrow_limit - checkouts.length
     end
 
-    def proxy_borrower?
-      false
-    end
-
-    def sponsor?
-      false
-    end
-
-    def checkouts
-      @checkouts ||= patron_info['loans'].map { |checkout| Checkout.new(checkout) }
+    def all_fines
+      # @all_fines ||= []
```

Comment:
Cruft?

Reply:
All these comments among fines were existing before this PR. They were commented out to make the model work in a first pass. The only reason this is coming up as a change is because I moved it within the file. They will be fixed in https://github.com/sul-dlss/mylibrary/issues/830